cmd/par/main.go

```diff
@@ -367,7 +366,8 @@ func main() {
 		err = errors.New("no command specified")
 	}
 	if err != nil || globalFlags.usage {
-		printUsageAndExit(name, allCommands, err)
+		printUsage(name, allCommands, err)
+		os.Exit(eSuccess)
```

Comment:
i think this should only return success if err is nil -- `printUsageAndExit` can probably condition on `err != nil` to figure out which error code to return